src/Notes/WC_Admin_Notes_Draw_Attention.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * WooCommerce Admin Draw Attention note provider
+ *
+ * Adds a note to the merchant's inbox
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Draw_Attention
+ */
+class WC_Admin_Notes_Draw_Attention {
+	const NOTE_NAME   = 'wc-admin-draw-attention';
+	const OPTION_NAME = 'woocommerce_onboarding_profile';
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		// Trigger this when the onboarding options are updated.
+		add_filter(
+			'update_option_' . self::OPTION_NAME,
+			array( $this, 'possibly_add_draw_attention_note' )
+		);
+	}
+
+	/**
+	 * Possibly add a draw attention note.
+	 */
+	public static function possibly_add_draw_attention_note() {
+		$onboarding_profile = get_option( self::OPTION_NAME );
```

Comment:
Instead of getting the option again here, the `update_option_{$option}` hook will pass `$old_value, $value, $option` as arguments to `possibly_add_draw_attention_note()` that we can check.

Reply:
Nice! Done.